clients/http-client-common/src/main/java/org/apache/servicecomb/http/client/event/CommonEventManager.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.http.client.event;
+
+import com.google.common.eventbus.EventBus;
+
+/**
+ * EventManager for chassis events
+ *
+ */
+public class CommonEventManager {
```

Comment:
no need add this class , use eventBus from outside paramter.

Reply:
Keep this public event class first, because it involves the inconsistency of multiple events after unification, and the event logic of the service center client module needs to be modified. To ensure reliability, retain this logic for the time being.

Reply:
done